add LIVEBOOK_FORCE_SSL env (#1064)
livebook-dev/livebook: README.md

    Enabled by default unless `LIVEBOOK_PASSWORD` is set. Set it to "false" to
    disable it.

  * LIVEBOOK_FORCE_SSL - force SSL connections and enable HSTS.
    Ensuring no data is ever sent via http, always redirect to https.

> 

Reviewer: Sorry, maybe there was some confusion. We don't need LIVEBOOK_FORCE_SSL anymore, only LIVEBOOK_FORCE_SSL_HOST. :)

Author: I've solved it.

    Enabled by default unless `LIVEBOOK_PASSWORD` is set. Set it to "false" to
    disable it.

>   * LIVEBOOK_FORCE_SSL_HOST - set a new host to redirect to if the request's scheme is `http`. 

Reviewer: ```suggestion
  * LIVEBOOK_FORCE_SSL_HOST - set a host to redirect to if the request is not over HTTP.
```

>     Defaults to nil.

Reviewer: ```suggestion
    Note it does not apply when accessing Livebook via localhost. Defaults to nil.
```

>   Ensuring no data is ever sent via http, always redirect to https.

Reviewer: ```suggestion
    Ensuring no data is ever sent via http, always redirect to https.
```
